alertlogic/dynapi.py

```diff
@@ -151,34 +168,43 @@ def call(self, session, url_args, json=None):
         :param url_args: dict with values to replace url parameters, see parse_args() for more info
         :param json: this will be passed adhoc to requests
         """
-        parsed_url = session.region.get_api_endpoint() + self.parse_url(url_args)
-        log.debug("calling requests: operation={} url={} json={}".format(self.operation, parsed_url, json))
-        return requests.request(self.operation, parsed_url, json=json, auth=session)
+        parsed_url = session.region.get_api_endpoint() + self.parse_url(
+            url_args)
+        log.debug("calling requests: operation={} url={} json={}".format(
+            self.operation, parsed_url, json))
+        return requests.request(
+            self.operation, parsed_url, json=json, auth=session)
+
 
 def substitute_path_args(path, args):
     parts = [part for part in path.lower().split("/") if len(part) > 0]
     substituted = ""
     for part in parts:
         if part.startswith(":"):
-            required_arg = part[1:] # removes ":" at the beginning of the string
+            required_arg = part[
+                1:]  # removes ":" at the beginning of the string
```

Comment:
what is the reason for such weird line break, it seems that pycodestyle is not complaining if it is on a single line.

Reply:
not fixed yet